### PR TITLE
add contributing

### LINKED ISSUE #151: Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Elli610**
closes #151

### CODE REVIEW

**Review by 0xSwapFeeder — CHANGES_REQUESTED** [review]
Formatting seems awkward, way to many blank lines I think.
Otherwise lgtm.

**Review by Ben-Rey — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+If you discover a bug, please create a [new issue](https://github.com/massalabs/massa-web3/issues/new?assignees=&labels=issue) on our GitHub repository.
+
+In your issue, please include a clear and concise description of the bug, any relevant code snippets, error messages, and steps to reproduce the issue.
+
+## Installation
+
+To start developing with massa-web3, you must install all the necessary dev dependencies. You can do so by running the following command:
+
+```sh
+
```
Comment (gregLibert) [inline CONTRIBUTING.md]:
```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+npm install
+
```
Comment (gregLibert) [inline CONTRIBUTING.md]:
```suggestion
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+This will install all the required packages listed in the package.json file, allowing you to update, fix, or improve massa-web3 in any way you see fit.
+
+## Contributing Code
+
+We welcome contributions in the form of bug fixes, enhancements, and new features.
+
+To contribute code, please follow these steps:
+
+1. Fork the massa-web3 repository to your own account.
+
```
Comment (gregLibert) [inline CONTRIBUTING.md]:
```suggestion
```

**Review by gregLibert — APPROVED** [review]

**Pull request merged**